cypress/components/button-minor/button-minor.test.js

```diff
@@ -94,6 +94,13 @@ context("Test for Button Minor component", () => {
       }
     );
 
+    it("when icon only with tooltip message, icon's position is absolute", () => {
+      CypressMountWithProviders(
+        <ButtonMinor iconType="bin" iconTooltipMessage="foo" />
```

Comment:
If you make the code change I suggested,the `iconTooltipMessage` prop won't be needed in this prop (and we can leave it out of the test description).